builder/repo_control.go

```diff
@@ -316,6 +317,12 @@ func (b *Builder) ListRepos() error {
 		if url == "" {
 			continue
 		}
+		// if file uri scheme, the repo can be accessed offline
+		if strings.Contains(url, "file://") {
+			b.repoURL[name] = strings.TrimPrefix(url, "file://")
```

Comment:
This check won't always give correct results.
Currently,  the `mixer repo` commands do not validate the urls. Users can even set offline repos without the "file://" prefix.